extensions/wikia/AdEngine/js/video/volumeControlHandler.js

```diff
@@ -0,0 +1,78 @@
+/*global define*/
+define('ext.wikia.adEngine.video.volumeControlHandler', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (doc, log, win) {
+	'use strict';
+	var logGroup = 'ext.wikia.adEngine.video.volumeControlHandler';
+
+	function init(ima, adContainer) {
+		var muteDiv,
+			speaker;
+
+		muteDiv = getMuteDiv(adContainer);
+		speaker = muteDiv.querySelector('.speaker');
+
+		muteDiv.addEventListener('click', function(e) {
+			e.preventDefault();
+			onAdMuteClick(ima, speaker);
+		});
+
+		ima.addEventListener(win.google.ima.AdEvent.Type.LOADED, function () {
+			unmute(ima, speaker);
+			muteDiv.classList.remove('hidden');
+		});
+		ima.addEventListener(win.google.ima.AdEvent.Type.COMPLETE, function () {
+			unmute(ima, speaker);
+			muteDiv.classList.add('hidden');
+		});
+	}
+
+	function getMuteDiv(adContainer) {
+		var muteDiv,
+			speaker;
+
+		muteDiv = adContainer.querySelector('.ima-mute-div');
+
+		if (!muteDiv) {
+			muteDiv = doc.createElement('div');
+			muteDiv.className = 'ima-mute-div';
+			muteDiv.classList.add('hidden');
+			speaker = doc.createElement('a');
+			speaker.className = 'speaker';
+			speaker.appendChild(doc.createElement('span'));
+			muteDiv.appendChild(speaker);
+			adContainer.appendChild(muteDiv);
+			log('volume control is added', log.levels.info, logGroup);
+		}
+
+		return muteDiv;
+	}
+
+	function onAdMuteClick(ima, speaker){
+		if (ima.adMuted) {
+			unmute(ima, speaker);
+		} else {
+			mute(ima, speaker);
+		}
+	}
+
+	function mute(ima, speaker) {
+		log('mute', log.levels.info, logGroup);
+		speaker.classList.add('mute');
+		ima.adMuted = true;
+		ima.adsManager.setVolume(0);
+	}
+
+	function unmute(ima, speaker) {
+		log('unmute', log.levels.info, logGroup);
+		speaker.classList.remove('mute');
+		ima.adMuted = false;
+		ima.adsManager.setVolume(1);
```

Comment:
How about setting it by default to `0.75`, so we'll follow [the spec](https://docs.google.com/presentation/d/1UtOBjbErX7m2EQFhvkTR8oUdpB1fxtVuM8pz4hI33FQ/edit#slide=id.g18603da2b6_0_22).